src/EditorFeatures/Core.Wpf/Suggestions/SuggestedActionsSourceProvider.cs

```diff
@@ -91,7 +93,11 @@ internal partial class SuggestedActionsSourceProvider : ISuggestedActionsSourceP
             if (textBuffer.IsInLspEditorContext())
                 return null;
 
-            return _optionService.GetOption(SuggestionsOptions.Asynchronous)
+            var document = textBuffer.CurrentSnapshot.GetOpenDocumentInCurrentContextWithChanges();
+            var experimentService = document?.Project.Solution.Workspace.Services.GetService<IExperimentationService>();
```

Comment:
Don't think it makes much of a difference here since the document can be null, but could be `GetRequiredService`